modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -0,0 +1,136 @@
+If an empty profile is not assigned to a hardware type group before registration, it will be assigned to group that best matches the product information available to it.
+
+For this procedure, you will require the system manufacturer name and product name for your terminal.
+
+.Procedure: Creating a Hardware Type group
+
+. Determine the hardware type group name.
+Prefix the name with [systemitem]``HWTYPE:``, followed by the system manufacturer name and product name, separated by a hyphen.
+For example:
++
+----
+HWTYPE:POSVendor-Terminal1
+----
++
+[IMPORTANT]
+====
+Only use colons (``:``), hyphens (``-``) or underscores (``_``) in hardware type group names.
+Spaces and other non-alphanumeric characters will be removed when the name is processed.
+====
++
+. In the {susemgr} {webui}, navigate to menu:Systems[System Groups], and click the btn:[Create Group] button.
+. In the [guimenu]``Create System Group`` dialog, create a new system group, using the hardware type group name you created earlier.
+
+
+== Assign and Configure the Saltboot formula for each hardware type group
+
+Each hardware type group must have the Saltboot formula applied.
+
+.Procedure: Assigning the Saltboot formula
+
+. Open the details page for your new hardware type group, and navigate to the [guimenu]``Formulas`` tab.
+. Select the [systemitem]``saltboot-formula`` and click btn:[Save].
+. Navigate to the new menu:Formulas[Saltboot] tab.
+. Configure the Saltboot formula according to the instructions in xref:retail:retail-formulas.adoc[].
+
+
+== Synchronize images to the branch server
+
+.Procedure: Synchronizing images to the branch server
+
+. On the {susemgr} server, run this command:
++
+----
+salt <branch_server_salt_id> state.apply image-sync
+----
+
+== Deploy images to the terminals
+
+.Procedure: Deploying images to the terminals
+
+. Power on your POS terminals.
+. The branch server will bootstrap the terminals according to the data you have provided.
+
+
+== Terminals based on {slsa}{nbsp}11 SP{nbsp}3
+
+POS Terminals based on {sls}{nbsp}11 SP{nbsp}3 can be deployed in much the same way as other terminals, with a few differences.
+
+* You must use the {slsa}{nbsp}11 template
+* {slsa}{nbsp}11 images need to be activated with the [systemitem]``SLES11SP4 i586`` and [systemitem]``SLES11SP4 SDK i586`` channels
+
+[IMPORTANT]
+====
+Ensure that {slsa}{nbsp}11 images are built on the {slsa}{nbsp}11 build host, and {slsa}{nbsp}12 images are built on the {slsa}{nbsp}12 build host.
+Building on the incorrect build host will cause your build to fail.
+====
+
+[WARNING]
+====
+If you are building images for {slsa}{nbsp}11 using profiles from an HTTPS git repository that uses TLS 1.0 or greater, it will fail.
+{slsa}{nbsp}11 does not support later versions of TLS.
+You will need to clone the repository locally in order to use it for building.
+====
+
+== Using a 32-bit default boot image
+
```

Comment:
This describes a configuration detail of image synchronization.
IMO it could be a subsection of "Synchronizing images to the branch server" above